### ISSUE TITLE
Add styling to main menu for current page

### ISSUE THREAD

**Issue by vavroom** [nicolas-steenhout/main#32]
Related to adding `aria-current` to the main nav.

[MDN says](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current):

>When you have a group of related elements, such as several links in a breadcrumb or steps in a multi-step flow, with one element in the group styled differently from the others to indicate to the sighted user that this is the current element within its group, the aria-current should be used to inform the assistive technology user what has been indicated via styling.

Should we add a visual styling? This isn't a breadcrumb, etc... But? Dunno. What do you think?

**Comment by digilou** [comment]
Definitely add a visual styling. Was waiting for the code bit.
